etcd/etcd-live-http-to-https-migration.md

```diff
@@ -1,73 +1,73 @@
+Create the `/etc/etcd2` directory, then copy the corresponding certificate and key there. Set permissions to secure the directory and key file:
 
 ```
-chown -R etcd:etcd /etc/etcd2/
-chmod 600 /etc/etcd2/*-key.pem
+$ chown -R etcd:etcd /etc/etcd2
+$ chmod 600 /etc/etcd2/*-key.pem
 ```
 
-Copy `ca.pem` CA certificate into `/etc/etcd2`.
+Copy the `ca.pem` CA certificate file into `/etc/etcd2` as well.
 
-Or copy your `ca.pem` CA certificate into `/etc/ssl/certs` directory and run `update-ca-certificates` script to update your certificates bundle. In this case CA will be available system wide and there will be not necessary to set CA path for each app.
+Alternatively, copy `ca.pem` into `/etc/ssl/certs` instead, and run the `update-ca-certificates` script to update the system certificates bundle. After doing so, the added CA will be available to any program running on the node, and it will not be necessary to set the CA path for each application.
 
-Repeat this step on rest of the member nodes.
+Repeat this step on the rest of the cluster members.
 
-### Using etcd Proxy
+### Using an etcd Proxy
 
-If you typically connect to an etcd cluster at a remote location, it's recommended to use this opportunity to configure an [etcd proxy][etcd proxy] that handles the remote connection logic and TLS termination, and reconfigure your apps to communicate through the proxy on localhost. In this case you have to generate client keypair (i.e. `client1.pem` and `client1-key.pem`) and follow the previous step.
+If you typically connect to a remote etcd cluster, this is a good time to configure an [etcd proxy][etcd proxy] that handles the remote connection and TLS termination, and to reconfigure your apps to communicate through the proxy on localhost. In this case, you must generate a client key pair (e.g. `client1.pem` and `client1-key.pem`) and repeat the *Copy Key Pairs* step above with the client key pair files.
 
-In addition it is necessary to modify your systemd [unit files][systemd-unit-file] or [drop-ins][drop-ins] which use `etcdctl` in `ExecStart*=` or `ExecStop*=` directives and replace `/usr/bin/etcdctl` with the `/usr/bin/etcdctl --no-sync`. This will force `etcdctl` to use proxy for all operations (not only members list).
+It is also necessary to modify your systemd [unit files][systemd-unit-file] or [drop-ins][drop-ins] which use `etcdctl` in `ExecStart*=` or `ExecStop*=` directives to replace the invocation of `/usr/bin/etcdctl` with `/usr/bin/etcdctl --no-sync`. This will force `etcdctl` to use the proxy for all operations.
 
-## Configure etcd Keypair
+## Configure etcd Key Pair
 
-Now we should configure etcd paths to the new certificates. Create `/etc/systemd/system/etcd2.service.d/30-certs.conf` [drop-in][drop-ins] with contents below:
+Now we will configure etcd to use the new certificates. Create a `/etc/systemd/system/etcd2.service.d/30-certs.conf` [drop-in][drop-ins] file with the following contents:
```

Comment:
Is this necessary if we put the cert(s) in the system bundle or only if we copied them to /etc/etcd2?